### ISSUE TITLE
Erroneous trailing quote

### ISSUE THREAD

**Issue by muffinresearch** [mozilla/addons-frontend#2698]
See https://github.com/mozilla/addons-frontend/blob/cc13364b2cca75e9b6ab131f0cead976b13a1e19/src/amo/components/SearchContextCard/index.js#L46

Extraction translation is here:

https://github.com/mozilla/addons-frontend/blob/cc13364b2cca75e9b6ab131f0cead976b13a1e19/locale/templates/LC_MESSAGES/amo.pot#L516

```
#: src/amo/components/SearchContextCard/index.js:45
msgid "%(count)s add-on found\""
msgid_plural "%(count)s add-ons found"
msgstr[0] ""
msgstr[1] "

```

Originally filed here: https://github.com/mozilla/addons/issues/436

**Comment by tofumatt** [comment]
Fixed in #2701

**Comment by SVSanta** [comment]
Please add some STR or the label "qa: not needed". Thanks!
